### ISSUE TITLE
The last version breaks the code references

### ISSUE THREAD

**Issue by abdonrd** [runem/vscode-lit-plugin#4]
With the last version (`v0.1.0`), now I can't hover references or cmd + click to "go to".

<img width="638" alt="screenshot 2019-02-22 at 18 27 08" src="https://user-images.githubusercontent.com/1007051/53259492-7aed1a00-36cf-11e9-9acb-82dc3acc22ef.png">

If I just disable the extension, it works again:

<img width="648" alt="screenshot 2019-02-22 at 18 23 58" src="https://user-images.githubusercontent.com/1007051/53259301-0b772a80-36cf-11e9-80f1-d41af80ec17b.png">

**Comment by abdonrd** [comment]
Friendly ping to @runem.

**Comment by runem** [comment]
I'll take a look at it ASAP  👍

**Comment by runem** [comment]
Hmm, strange, I have problems reproducing it. Would it be possible for you to send me the content of `tsserver.log`? You can find it by opening the command palette (`cmd + shift + p` on mac) and searching for "Open TS server log". I really only need to see entries with `"command":"definitionAndBoundSpan"` and potential stack traces.

**Comment by abdonrd** [comment]
Sure! Here it is! [tsserver.log](#)

I have used a new project based on [Polymer/pwa-starter-kit](https://github.com/Polymer/pwa-starter-kit) as example.

**Comment by runem** [comment]
I have now fixed the problem in version `0.1.1` :tada: 

Thank you so much for taking your time to report it! 

If you are interested in knowing what the issue was, here is a description of the problem and what I did to solve it. 

It was easy to reproduce it after cloning the pwa-starter-kit. I saw in the logs that the file `@polymer/app-layout/app-drawer/app-drawer.d.ts'` gave my plugin problems. Then I found that this element (including the other @polymer elements) are defined as interfaces. I haven't taken interfaces into account and unfortunately it crashed, but now I treat them the same ways as classes. I also added a failsafe so my plugin wont crash everything again, sorry :-) 

In addition I can see that there are some attributes in the pwa-starter-kit that aren't recognized. I'll look into this matter but attributes like `<a ?selected ...` should be added to `globalHtmlAttributes` and attributes like `<app-header effects="waterfall"` ("effects" doesn't seem to be defined as a property and therefore my plugin can't find it) is a library challenge because the library will need to ship some metadata (see your other issue at https://github.com/runem/vscode-lit-plugin/issues/1) 

Edit: I see that "effects" is added through `import '@polymer/app-layout/app-scroll-effects/effects/waterfall.js'`. Then it should be added to the `globalHtmlAttributes` configuration.
